packages/cardhost/app/templates/components/card-name-dialog.hbs

```diff
@@ -0,0 +1,16 @@
+<Dialog open={{true}} @title={{this.title}} @mode={{@mode}} @icon={{@icon}}>
+  <p class="dialog--message">
+    Give your card a meaningful name
+  </p>
+  <TextField
+    @id="card__name"
+    @placeholder="i.e. Millenial Puppies"
```

Comment:
```suggestion
    @placeholder="i.e. My Event Name"
```

We should avoid 'Millennial Puppies' and Ember Meetup things for our examples, and take inspiration from the cards that are shown on the website now.